README.md

```diff
@@ -2,26 +2,21 @@
 
 # Project Name
 
-> Description the project.
-
+> Hello Microverse
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML and CSS
 
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
 
-
 ## Getting Started
 
 **This is an example of how you may give instructions on setting up your project locally.**
 **Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
 
```

Comment:
Please correct the info inside the `Get Started` section by adding information about how to clone and run your project locally to make your Readme file look more professional.